build.sbt

```diff
@@ -15,16 +18,14 @@ inThisBuild(
         )
       ),
     scalaVersion := scala212,
-    scalafixDependencies +=
-      "com.github.liancheng" %% "organize-imports" % "0.5.0",
```

Comment:
Was integrated in Scalafix 0.11.0

See https://github.com/scalacenter/scalafix/blob/v0.11.0/docs/rules/OrganizeImports.md